perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -1928,6 +1959,37 @@ public Application getApplicationById(PerunSession sess, int appId) throws Regis
 
 	}
 
+	@Override
+	public List<Application> getApplicationsForGroup(PerunSession userSession, Group group, List<String> state, String dateFrom, String dateTo) throws PerunException {
+		groupsManager.checkGroupExists(userSession, group);
+
+		//Authorization
+		if (!AuthzResolver.authorizedInternal(userSession, "getApplicationsForGroup_Vo_List<String>_policy", Collections.singletonList(group))) {
+			throw new PrivilegeException(userSession, "getApplicationsForGroup");
+		}
+		if (state == null || state.isEmpty()) {
+			// list all
+			try {
+				return jdbc.query(APP_SELECT + " where a.group_id=? and a.created_at::date >= date ? and a.create_at::date <= date ? order by a.id desc", APP_MAPPER, group.getId(), dateFrom, dateTo);
```

Comment:
This SQL also seems to be incorrect. This will again work only if you use LocalDates.
```suggestion
				return jdbc.query(APP_SELECT + " where a.group_id=? and a.created_at::date >= ? and a.created_at::date <= ? order by a.id desc", APP_MAPPER, group.getId(), dateFrom, dateTo);
```

Reply:
I do not understand. AFAIK the SQL is correct and works (tested on command line).

Reply:
The column `create_at` is incorrect. Also, the notation `date ?` doesn't work in JDBC. I have run it, and it just throws `PSQLException: ERROR: syntax error at or near "$2"`.

Rewritting it to `date(?)` or `?::date` seemed to fix it.

Reply:
Changed to 'date(:param)'